### PR TITLE
Add block $bl snippet

### PR DESCRIPTION
**Pull request by ourmaninamsterdam**
This PR is to help get some feedback on the structure/purpose of this snippet.

The idea is to have a generic block component that outputs a simple 2 column layout. The content within these can be anything.

### CODE REVIEW

**Comment by tomas-stefano** [comment]
I like the idea, lgtm the idea.

File: spec/mastalk_spec.rb [hunk 1]
```diff
@@ -157,4 +157,41 @@
...
+  end
+
+  context 'collapsable section' do
+    let(:source) { '$- -$' }
+
+    let(:expected) do
+      %(<div class="collapsible-section">\n  \n\n</div>\n)
+    end
+
+    it 'pre-processes correctly' do
```
Comment (tomas-stefano) [inline spec/mastalk_spec.rb]: Can we put a better message? E.g 'returns a collapsible section' or something related.

File: spec/mastalk_spec.rb [hunk 2]
```diff
@@ -157,4 +157,41 @@
...
+  end
+
+  context 'collapsable header' do
+    let(:source) { '$= Content =$' }
+
+    let(:expected) do
+      %(<div class="collapsible">\n  <button class="unstyled-button">\n    <span class="icon icon--toggle"></span>\n    <span class=\"visually-hidden js-collapsable-hidden\"></span>\n     Content \n  </button>\n</div>\n)
+    end
+
+    it 'pre-processes correctly' do
```
Comment (tomas-stefano) [inline spec/mastalk_spec.rb]: Here too.

File: spec/mastalk_spec.rb [hunk 3]
```diff
@@ -157,4 +157,41 @@
...
+      expect(subject.to_html).to eq(expected)
+    end
+  end
+
```
Comment (tomas-stefano) [inline spec/mastalk_spec.rb]: You can remove this line.

**Comment by ourmaninamsterdam** [comment]
@tomas-stefano @dougdroper have made the suggested mods.

**Comment by dougdroper** [comment]
:+1:

**Comment by tomas-stefano** [comment]
Many thumbs up on that one! :+1: :+1: 
many-thumbs-up

**Pull request merged**